plugins/inputs/kafka_consumer/README.md

```diff
@@ -80,12 +80,12 @@ and use the old zookeeper connection method.
   ## larger messages are dropped
   max_message_len = 1000000
 
-  ## Maximum messages to read from the broker that have not been written by an
-  ## output.  For best throughput set based on the number of metrics within
-  ## each message and the size of the output's metric_batch_size.
+  ## Maximum messages to read from the broker that have not been written by
+  ## an output. For best throughput, set max_undelivered_messages to at or
+  ## above the metric_batch_size.
   ##
   ## For example, if each message from the queue contains 10 metrics and the
-  ## output metric_batch_size is 1000, setting this to 100 will ensure that a
+  ## output metric_batch_size is 1000, setting this to 1000 will ensure that a
```

Comment:
I don't think your edited text is correct. Telegraf performs best when max_undelivered_messages is the metric_batch_size divided by the number of metrics in each message.

Reply:
@reimda,

your comments come across as two different options:

* Your first comment (Oct 4), which I think I agree with says `max_undelivered_messages = metric_batch_size / # of metrics`. 
* Your later comment (Oct 7) reads like `metric_batch_size = max_undelivered_messages + # of metrics`

My understanding is that the first option is the more correct recommendation.

Thoughts?